examples/threadExample/src/main/java/io/realm/examples/threads/AsyncQueryFragment.java

```diff
@@ -0,0 +1,190 @@
+/*
```

Comment:
You can't have them in parallel, because the example will not compile (new API will not be available).